bootstrap/handlers/httpserver.go

```diff
@@ -156,7 +168,70 @@ func (b *HttpServer) BootstrapHandler(
 		}()
 
 		b.isRunning = true
-		err := server.ListenAndServe()
+
+		var ln net.Listener
+		switch bootstrapConfig.Service.SecurityOptions["Mode"] {
+		case "zerotrust":
+			secretProvider := container.SecretProviderExtFrom(dic.Get)
+			var zitiCtx ziti.Context
+			var ctxErr error
+			if secretProvider != nil {
+				jwt, jwtErr := secretProvider.GetSelfJWT()
+				if jwtErr != nil {
+					lc.Errorf("could not load jwt: %v", jwtErr)
+				}
+				lc.Info("using zerotrust - look at you go: " + jwt)
```

Comment:
Should we be logging the JWT??

Reply:
definitely not.  :) i had left it here to aid with debugging. i planned to take it out after things were working. since they're working now, i'll remove it